satellite/analytics/service.go

```diff
@@ -15,7 +15,10 @@ const (
 	eventSignedIn             = "Signed In"
 	eventProjectCreated       = "Project Created"
 	eventAccessGrantCreated   = "Access Grant Created"
+	eventAccountVerified      = "Account Verified"
```

Comment:
we should make sure to use consistency in our naming here. Either all of these should have `event` in front of them, or none of them should have `event` in front of them

Reply:
Done